%_______________________________________________________________________________
%class
%_______________________________________________________________________________
%\documentclass[a4paper,11pt,onecolumn,final,german,openbib]{scrbook}
\documentclass[a4paper,11pt,oneside,final,english,toc=bib,draft]{scrbook}

% fonts

\usepackage{fontspec}
\setmainfont{Libre Baskerville} % Libre Baskerville - Palatino Linotype - Constantia
\setsansfont{Helvetica} % Arial
\setmonofont{Consolas} % Hack

\usepackage[OT1]{eulervm}
% \usepackage{mathdesign}

%_______________________________________________________________________________
% page borders
%_______________________________________________________________________________
\addtolength{\headheight}{2cm}
%\addtolength{\topmargin}{2cm}
\setlength{\oddsidemargin}{1.0cm}
\setlength{\evensidemargin}{0.5cm}
\setlength{\textwidth}{14.3cm}
\setlength{\parindent}{0mm}

%_______________________________________________________________________________
% packages
%_______________________________________________________________________________
% \usepackage{german}
\usepackage{babel}[uk-english] % english hyphenation
% \usepackage{textcomp}
\usepackage{csquotes} % defines \enquote for enquoting text

\usepackage{mathtools} % defines common math symbols and tools
\usepackage{amssymb,amsfonts} % more math symbols and fonts
\usepackage{physics} % all-purpose physics and math notation
\usepackage[separate-uncertainty=true,per-mode=power,group-separator={,}]{siunitx} % numbers with units
% \usepackage[version=4]{mhchem} % chemical equations
% \usepackage{dsfont} % math font for double-stroked letters
% \usepackage{slashed} % math font for slashed symbols

\usepackage{graphicx} % \includegraphics and \textcolor
\usepackage{xcolor} % define common colors
% \usepackage{subfigure}
\usepackage{float} % 
\usepackage{caption} % configure caption style for figures and tables
\usepackage{subcaption} % subfigures
\usepackage{placeins} % defines \FloatBarrier, which forces floats to be rendered before
\usepackage{enumitem} % better lists

% \usepackage{url}
% \usepackage{enumerate}

\usepackage{booktabs} % prettier tables
% \usepackage{multirow}

% \usepackage{appendix} % appendix section

\usepackage{todonotes} % insert TODO notes

\usepackage[style=ieee]{biblatex} % ,sorting=none defernumbers=true style=h-physrev3/ieee
\bibliography{thesis}


\graphicspath{ {./images/} }

\usepackage[draft=false]{hyperref} % links inside document
\hypersetup{
  colorlinks=true,
  linkcolor=blue,
  citecolor=teal
}

\captionsetup[figure]{labelfont=bf,format=plain,labelsep=newline} % ,labelsep=period ,margin=1cm
\captionsetup[table]{labelfont=bf,format=plain,labelsep=newline} % ,labelsep=period ,margin=1cm ,labelsep=newline

\renewcommand{\arraystretch}{1.2} % make distance between table rows a bit bigger

\newcommand{\figwidth}{11.5cm} % default width for all figures

\parindent10mm

% \makesavenoteenv{table}

\usepackage{mgscience}

%_______________________________________________________________________________
% bold fonts for headings
%_______________________________________________________________________________
\font\afont=cmssbx10 scaled \magstep5     % for the title
\font\bfont=cmssbx10 scaled \magstep4     % for chapter headings
\font\cfont=cmssbx10 scaled \magstep3
\font\dfont=cmssbx10 scaled \magstep2     % for section headings and author name
\font\efont=cmssbx10 scaled \magstephalf

%_______________________________________________________________________________
% index depth
%_______________________________________________________________________________
\setcounter{secnumdepth}{3}
\setcounter{tocdepth}{3}

%_______________________________________________________________________________
% new commands
%_______________________________________________________________________________
% \newcommand{\demi}{\frac{1}{2}}

%_______________________________________________________________________________
% renewed commands
%_______________________________________________________________________________
% \renewcommand{\topfraction}{1.}       % this is important for figure placement
% \renewcommand{\bottomfraction}{1.}
\makeatletter
\renewcommand\paragraph{\@startsection{paragraph}{4}{\z@}%
  {-3.25ex\@plus -1ex \@minus -.2ex}%
  {1.5ex \@plus .2ex}%
  {\normalfont\normalsize\bfseries}
}
\makeatother

%_______________________________________________________________________________
% special words, hyphenation
%_______________________________________________________________________________
% \hyphenation{Ba-che-lor-ar-beit}

\pagestyle{empty}
\pagestyle{headings}
%for changing the style on a specific page use \thispagestyle{e.g., empty}

%_______________________________________________________________________________
%_______________________________________________________________________________
\begin{document}
\pagenumbering{roman}

%_______________________________________________________________________________
\begin{titlepage}
  \vspace*{6mm}
  \begin{center}
     {\afont Comparison of 3-dimensional genome structures based on single cell Hi-C data}
     \\[3.5cm]
     {\large von}
     \\[3.5cm]
     {\dfont Moritz Gmeiner}
     \\[1.5cm]
     {\dfont Supervisor: PD Dr. Peter Virnau}
     \\[2cm]
     % {\large Bachelorarbeit in Physik \/\\
     %    vorgelegt dem Fachbereich Physik, Mathematik und Informatik (FB 08) \/\\
     %    der Johannes Gutenberg-Universität Mainz \/\\
     %    am 1. April 2012}
     {\large Bachelor Thesis in Physics \/\\
        presented to the faculty physics, mathematics, and computer science (FB 08) \/\\
        of the Johannes Gutenberg University Mainz \/\\
        \today}
   \end{center}
   \vfill
   1. Reviewer: PD Dr. Peter Virnau \\	
   2. Reviewer: Prof. Dr. Friederike Schmid \\
   \vfill
\end{titlepage}

\thispagestyle{empty}
Ich versichere, dass ich die Arbeit selbstständig verfasst und keine 
anderen als die angegebenen Quellen und Hilfsmittel benutzt sowie 
Zitate kenntlich gemacht habe.
\\[1cm]
\includegraphics[height=1.5cm]{Unterschrift.png} \\
% \vspace{-1cm} \\
Moritz Gmeiner \hfill Mainz, \today
\vfill
\noindent 
Moritz Gmeiner\\
KOMET\\
Institut für Physik\\
Staudingerweg 7\\
Johannes Gutenberg-Universität
D-55099 Mainz\\
{ \texttt{mgmeiner@students.uni-mainz.de} }

%_______________________________________________________________________________
% \renewcommand\contentsname{Inhaltsverzeichnis}
% \renewcommand\figurename{Abbildung}
% \renewcommand\tablename{Tabelle}

\tableofcontents
\clearpage

\mainmatter
\sloppy

%_______________________________________________________________________________
% \include{...}
\chapter{Introduction} % (fold)
\label{cha:introduction}

\section{DNA and the genome} % (fold)
\label{sec:dna_and_genome}

DNA is the most important long-term information storage in biological systems. It is so fundamental that it is a shared property of all known life and even some non-living objects like viruses\footnote{The question whether or not viruses are alive or not is somewhat dependent on the definition of life and has not been finally decided (and probably never will), but here they shall be considered non-living.}. In eukaryotes, most of the genome\footnote{The term \enquote{genome} typically refers to the entirety all of the DNA in the cell, including e.g. mitochondrial and plastid DNA. For simplicity, in this work the term genome will include only the DNA in the nucleus, as it is the main and for this work only relevant part} is stored away in the cell's nucleus, a specialized compartment that is separated from the rest of the cell by the nuclear envelope. It is split up into multiple chromosomes, linear, i.e. non-circular, chains of DNA of varying length. The number of of different chromosomes in each cell is dependent on the particular species, for example humans have 23 different chromosomes while mice, the primary subject of this work, have 20. Different species and cell types also vary in the number of copies of each chromosome they carry: haploid cells for example carry only a single copy of each chromosome, while diploid cells have two copies of each chromosome. To fit into the nucleus, the genome needs to be highly packaged: the length of the human genome for example is about \(\SI{2}{\metre}\), while the cell nucleus is merely \(5\)-\(8\) \(\si{\micro\metre}\) in diameter. This packaging occurs at different levels, like histones and the 30-nm fibre. The process of how exactly DNA is being packaged is a very important research topic, as it is involved in transcription initiation, the first step of making proteins from DNA information: By making certain regions of DNA more or less accessible, specific parts of the genome can either be activated or silenced, which is an important way of regulating gene expression. \cite{uzman_essential_2010}

% section dna_and_genome (end)

\section{Hi-C} % (fold)
\label{sec:hi_c}

Hi-C is a method of determining the three-dimensional structure of an entire genome utilising a combination of proximity-based ligation and DNA deep sequencing. It is a continuation of a number of previous chromosome conformation capture methods (3C, 4C, 5C) with the explicit goal of moving from targeting only specific genetic loci to obtaining a genome-wide map of interactions.

First, the DNA is cross-linked using formaldehyde, making connections between two strands of DNA that are in close proximity. The DNA is then digested by a restriction endonuclease that leaves a 5' staggered end which is filled with biotin-marked residues and ligated under conditions that favour ligation of those cross-linked DNA fragments. The DNA is sheared and selected for the biotin-marked cross-linked DNA fragments. These fragments are then deep sequenced and mapped back onto the genome, giving a list of contacts between positions on the genome.\cite{lieberman-aiden_comprehensive_2009}.

% section hi_c (end)

\section{Simulation of genome structure} % (fold)
\label{sec:simulation_of_genome_structure}

After obtaining a contact map of a particular genome using Hi-C, it can then be used to try to reconstuct the three-dimensional structure of the genome in the original physical cell that was mapped. Various approaches have been used for this task, including simulated annealing\cite{stevens_3d_2017} or polymeric Gō-models\cite{wettermann_minimal_2020}. One specific area of interest is the comparability of these reconstructed genomes structures across different cells of the same cell line, both as a measure of how well the model can reconstuct the original structure as well as to determine similarities and differences in the genome structure between the actual physical cells. In particular finding certain features that are shared across multiple cells might provide further insight in how exactly the folded structure of the genome influences transcription and, in reverse, how nature might utilise the structure of the genome as a mechanism to influence transcription. To this end, Hi-C data sets extracted from eight haploid mouse embryonic stem cells, obtained from (Stevens et al.\cite{stevens_3d_2017}), were used to reconstruct the genome structure using a generic polymer model and then compared on both the genome level as well as the chromosomal level. For this first an analysis of the simulation results of each cell for itself will be made and compared to the results of (Wettermann et al.\cite{wettermann_minimal_2020}). Then various statistical tools will used to compare the obtained structures of the different cells, first on an entire genome level and then on an individual chromosome level.

% section simulation_of_genome_structure (end)

% \section{Mathematical Tools} % (fold)
% \label{sec:mathematical_tools}

% \subsection{RMSD} % (fold)
% \label{sec:rmsd}

% The root-mean-square deviation (RMSD) is a mathematical tools for comparing how similar two ordered sets of 3-dimensional points of equal length are and is given for the two sets \( \set{x_i} \) and \( \set{y_i} \) by the formula

% \[
%   \text{RMSD} = \sqrt{ \frac{n} \sum_i \norm{x_i - y_i}^2 }
% \]

% While there are certain ready-made tools to calculate RMSDs of trajectories, for example in VMD's\cite{humphrey_vmd_1996} RMSD Trajectory tool, for this work a more custom approach was chosen in order to have more flexibility. The code for RMSD calculation can be found in the \verb|tools/rmsd.py| file, in particular in the \verb|rmsd(pos1, pos2| function, which calculates the RMSD between two point sets using scipy's\cite{virtanen_scipy_2020} \verb|scipy.spatial.procrustes| function.

% % section rmsd (end)

% section mathematical_tools (end)



%_______________________________________________________________________________
% \chapter{Main Part}

% Die typische Gliederung einer Bachelorarbeit könnte so aussehen, 
% wie im folgenden dargestellt. 
% \medskip

% Verwenden Sie aussagekräftige Kapitelüberschriften, also zum 
% Beispiel {\em Aufbau eines Teilchenbeschleunigers} statt 
% {\em Versuchsaufbau}.


% chapter introduction (end)

%_______________________________________________________________________________


\include{tex/simulation}


%_______________________________________________________________________________


\include{tex/comparison_of_cells}


%_______________________________________________________________________________


\include{tex/individual_chromosomes}


%_______________________________________________________________________________


\include{tex/conclusion}


%_______________________________________________________________________________


\printbibliography


%_______________________________________________________________________________
\appendix

% \chapter{Appendix}
\chapter{Tables and Figures} % (fold)
\label{cha:tables_and_figures}

\medskip

\begin{table}[H]
\centering
  \caption{Number of beads in each chromosome in the simulation. Each bead represents \(\SI{100}{kbp}\)}
  \label{tab:chrom_lengths}
  \begin{tabular}{c c c c c c c c c c c}
   \toprule
    Chrom & 1 & 2 & 3 & 4 & 5 & 6 & 7 & 8 & 9 & 10 \\
    Length & 1924 & 1791 & 1570 & 1534 & 1488 & 1466 & 1424 & 1263 & 1215 & 1275 \\
  \midrule
    Chrom & 11 & 12 & 13 & 14 & 15 & 16 & 17 & 18 & 19 & X \\
    Length & 1189 & 1171 & 1174 & 1218 & 1010 & 952 & 919 & 876 & 584 & 1671 \\
  \bottomrule
  \end{tabular}
\end{table}

\begin{table}[ht]
\centering
  \caption{Some statistics about each simulated cell. The pot(ential) energy \% deviation describes the standard deviation of the potential energy divided by the mean potential energy over all frames except for the first 5. For both bonds and contacts the mean and 99.73th percentile are of the distances between the \emph{predefined} bonds and contacts, i.e. only the ones were set as input before the simulation, not the ones arising in the simulation.}
  \label{tab:simulation_pe_dists}
  \sisetup{ table-alignment-mode=none }
  \begin{tabular}{c @{\phantom{abc}} S @{\phantom{abc}} S @{\phantom{abc}} S @{\phantom{abc}} S @{\phantom{abc}} S}
  \toprule
     & & \multicolumn{2}{c}{bond lengths} & \multicolumn{2}{c}{contact lengths} \\
   \cmidrule(rl){3-4}
   \cmidrule(rl){5-6}
    {Cell} & \parbox{3cm}{\centering pot energy \\ \% deviation} & {mean} & \parbox{3cm}{\centering 99.73th \\ percentile} & {mean} & \parbox{3cm}{\centering 99.73th \\ percentile} \\
  \midrule
    1 & 8.51\% & 1.13 & 1.98 & 1.62 & 2.72 \\
    2 & 0.27\% & 1.10 & 1.71 & 1.61 & 2.42 \\
    3 & 0.67\% & 1.14 & 1.88 & 1.68 & 2.52 \\
    4 & 1.83\% & 1.14 & 2.08 & 1.61 & 2.46 \\
    5 & 8.63\% & 1.11 & 1.81 & 1.62 & 2.55 \\
    6 & 0.94\% & 1.16 & 2.07 & 1.61 & 2.38 \\
    7 & 0.29\% & 1.10 & 1.58 & 1.59 & 2.09 \\
    8 & 0.28\% & 1.08 & 1.56 & 1.60 & 2.25 \\
  \bottomrule
  \end{tabular}
\end{table}

\begin{table}[ht]
\centering
  \caption{caption}
  \label{tab:contact_capture}
  \sisetup{ table-alignment-mode=none }
  \begin{tabular}{c S S S}
  \toprule
    Cell & \parbox{3cm}{\centering contacts prespecified} & \parbox{3cm}{\centering contacts after simulation} & \parbox{3cm}{\centering \% of contacts specified} \\
  \midrule
    1 & 48962 & 932831 & 5.2\% \\
    2 & 32243 & 558981 & 5.8\% \\
    3 & 19112 & 602439 & 3.2\% \\
    4 & 35514 & 685314 & 5.2\% \\
    5 & 31180 & 592569 & 5.3\% \\
    6 & 32862 & 358721 & 4.8\% \\
    7 & 21126 & 358721 & 5.9\% \\
    8 & 17581 & 320656 & 5.5\% \\
  \bottomrule
  \end{tabular}
\end{table}

\begin{figure}[ht]
\centering
  \includegraphics[width=\figwidth]{potential_energy_cell1_1.png}
  \caption{Potential energies for the frames of the first repeat simulation of cell 1.}
  \label{fig:potential_energy_cell1_1}
\end{figure}

\begin{figure}[ht]
\centering
  \includegraphics[width=\figwidth]{potential_energy_cell1_2.png}
  \caption{Potential energies for the frames of the second repeat simulation of cell 1.}
  \label{fig:potential_energy_cell1_2}
\end{figure}

\begin{figure}[ht]
\centering
  \includegraphics[width=\figwidth]{hic_vs_sim_scc.png}
  \caption{SCCs between the original Hi-C contact matrices and the contact matrices of the simulated cell genomes.}
  \label{fig:hic_vs_sim_scc}
\end{figure}

\begin{figure}[ht]
\centering
  \includegraphics[width=\textwidth+0.5cm]{rmsd_last_vs_avg.png}
  \caption{RMSD of each frame to both the last frame of the simulation and the average trajectory of the cell for each cell. The average trajectories were generated from a subset of frames from each cell that represents the ground state configuration of that cell in the simulation and is described in more detail in section~\ref{sec:rmsd_between_cells}.}
  \label{fig:rmsd_last_vs_avg}
\end{figure}

% chapter tables_and_figures (end)

\chapter{Used Software} % (fold)
\label{cha:used_software}

Figures were created using matplotlib\cite{j_d_hunter_matplotlib_2007} and seaborn\cite{waskom_seaborn_2021}.

\begin{table}[H]
\centering
\label{tab:used_software}
\caption{Software used for simulation and data analysis. Two systems were used, the top one a Manjaro Linux (based on Arch Linux) system and the bottom one a Microsoft Windows system with Anaconda}
  \begin{tabular}{l @{\phantom{abc}} l @{\phantom{abc}} l}
  \toprule
    Package & Version & Package Source \\
  \midrule
    HOOMD-blue & 1.9.7 & built from source \\
    numpy & 1.22.2 & arch repo \\
    scipy & 1.8.0 & arch repo \\
    pandas & 1.4.0 & arch repo \\
    matplotlib & 3.5.1 & arch repo \\
    gsd & 2.5.1 & pip \\
    vmd & 1.9.3 & \url{https://www.ks.uiuc.edu/Research/vmd/} \\
  \midrule
    numpy & 1.20.3 & anaconda \\
    scipy & 1.7.1 & anaconda \\
    pandas & 1.3.4 & anaconda \\
    matplotlib & 3.4.3 & anaconda \\
    gsd & 2.5.1 & pip \\
    hicreppy & 771cf72 & github \\
  \bottomrule
  \end{tabular}
\end{table}

\todo{packages and versions for linux system}


% chapter used_software (end)

\include{tex/renderings}

% chapter renderings_of_simuated_cells (end)

%_______________________________________________________________________________
% \chapter{Weiterführende Details zur Arbeit} % (fold)
% \label{cha:weiterführende_details_zur_arbeit}

% Manch wichtiger Teil Ihrer tatsächlichen Arbeit ist zu technisch 
% und würde den Hauptteil des Textes unübersichtlich machen, 
% beispielsweise wenn es um die Details des Versuchsaufbaus in einer 
% experimentellen Arbeit oder um den für eine numerische Auswertung 
% verwendeten Algorithmus geht. Dennoch ist es sinnvoll, entsprechende 
% Beschreibungen in einem Anhang Ihrer Bachelorarbeit aufzunehmen. 
% Insbesondere für zukünftige Arbeiten, die an Ihre Bachelorarbeit 
% anschließen, sind dies manchmal hilfreiche Informationen.

% % chapter weiterführende_details_zur_arbeit (end)

%_______________________________________________________________________________
% \chapter{Thanking} % (fold)
% \label{cha:thanking}

% ... an wen auch immer. Denken Sie an Ihre Freundinnen und Freunde, 
% Familie, Lehrer, Berater und Kollegen.

% chapter thanking (end)

%_______________________________________________________________________________
% \chapter{References} % (fold)
% \label{cha:references}

% Machen Sie genaue Angaben, so dass die verwendeten Literaturstellen 
% eindeutig identifiziert und aufgefunden werden können.
% Bei Lehrbüchern cite{Weinberg:1995mt} ist es sinnvoll, 
% den Titel anzugeben, eventuell auch die Ausgabe. Bei Artikeln in 
% Fachzeitschriften cite{Moch:2001zr} ist es üblich, nur die 
% gebräuchlichen Abkürzungen für den Titel der Zeitschrift, Band, 
% Erscheinungsjahr und Seite anzugeben. Unter Umständen kann es auch 
% sinnvoll sein, im Internet aufgefundene Informationsquellen anzugeben, 
% zum Beispiel für Software cite{LoopTools} oder zu den Details von 
% Ergebnissen großer experimenteller Kollaborationen. Es ist 
% selbstverständlich, dass Sie auch Bachelor- cite{BA:Freund}, 
% Diplom- oder Doktorarbeiten angeben, wenn Sie diese in Ihrer eigenen 
% Arbeit verwendet haben.
% \medskip

% Im folgenden Beispiel werden die in der Datei \texttt{h-physrev3.bst} 
% enthaltenen Anweisungen als Stilvorlage verwendet. Andere 
% Möglichkeiten für die Gestaltung eines Literaturverzeichnisses 
% findet man im Internet: \url{http://janeden.net/bibliographien-mit-latex}.

% \renewcommand{\bibname}{Literaturverzeichnis} 
% \bibliographystyle{h-physrev3}
% \begin{thebibliography}{99}

% %\cite{thepnews}
% \bibitem{EbelBliefert}
% H.\ F.\ Ebel, C.\ Bliefert, 
%   ``Bachelor-, Master- und Doktorarbeit: Anleitungen für den 
%   naturwissenschaftlich-technischen Nachwuchs,''
%   Wiley-VCH, Weinheim (2009). 
% \bibitem{thepnews}
%   S.~Becker, D.~Götz, C.~Reuschle, C.~Schwan, S.~Weinzierl,  
%   \url{http://wwwthep.physik.uni-mainz.de/site/news/168/}.

% %\cite{Weinberg:1995mt}
% \bibitem{Weinberg:1995mt}
%   S.~Weinberg,
%   ``The Quantum theory of fields. Vol. 1: Foundations,''
%   Cambridge, UK: Univ. Pr. (1995) 609 p.

% %\cite{Moch:2001zr}
% \bibitem{Moch:2001zr}
%   S.~Moch, P.~Uwer, S.~Weinzierl,
%   %``Nested sums, expansion of transcendental functions and multiscale 
%   % multiloop integrals,''
%   J.\ Math.\ Phys.\  {43 } (2002)  3363-3386.
%   [hep-ph/0110083].

% %\cite{LoopTools}
% \bibitem{LoopTools}
%   T.~Hahn, 
%   ``The LoopTools Site,''
%   \url{http://www.feynarts.de/looptools/}.

% %\cite{BA:Freund}
% \bibitem{BA:Freund}
%   B.~Freund, 
%   Bachelorarbeit, Johannes Gutenberg-Universität Mainz, 2012.

% \end{thebibliography}

% chapter references (end)


\end{document}