%!TEX root = ../thesis

\chapter{Individual Chromosomes} % (fold)
\label{cha:individual_chromosomes}

\section{Comparison of chromosomes in the entire genome simulations} % (fold)
\label{sec:comparison_of_chromosomes_in_the_entire_genome_simulations}

The comparison of the simulated structure between cells can be made on the chromosome level instead of the entire genome level. The results can be seen in Table~\ref{tab:mean_chrom_cross_rmsd}. While the mean RMSDs are significantly smaller than for the entire-genome case, they are still high enough to suggest no real similarity between the chromosomes across all cells, and the decrease can be explained simply by the smaller number of constraints when aligning the two trajectories. Considering even the minimum RMSD of each chromosome is never smaller than \(4.7\), it can be concluded that very likely there is no significant similarity between each chromosome across the different cells.

\begin{table}[ht]
\centering
  \caption{Mean and minimum of RMSDs of a particular chromosome pairwise between the averaged trajectories of all cells, excluding self-comparisons.}
  \label{tab:mean_chrom_cross_rmsd}
  \begin{tabular}{l @{\phantom{abc}} S S S S S S S S S S}
  \toprule
    Chrom & {1} & {2} & {3} & {4} & {5} & {6} & {7} & {8} & {9} & {10} \\
    Mean RMSD & 9.6 & 8.1 & 8.4 & 8.3 & 8.1 & 7.5 & 8.3 & 8.0 & 8.1 & 7.9 \\
    Min RMSD  & 6.8 & 6.5 & 5.6 & 5.3 & 6.1 & 5.9 & 5.9 & 6.0 & 6.3 & 4.9 \\
  \midrule
    Chrom & {11} & {12} & {13} & {14} & {15} & {16} & {17} & {18} & {19} & {X} \\
    Mean RMSD & 7.8 & 8.2 & 8.6 & 8.4 & 7.1 & 7.4 & 7.3 & 6.9 & 7.0 & 8.7 \\
    Min RMSD  & 6.6 & 5.6 & 5.6 & 5.6 & 5.4 & 5.3 & 5.5 & 5.0 & 4.7 & 6.7 \\
  \bottomrule
  \end{tabular}
\end{table}

An analysis that is somewhat complementary to the above is to study the relative chromsome interaction strengths in each cells, that is how strongly each pair of chromosomes interacts with each other. A naive way to calculate this interaction strength is to simply count the number of beads between each pair of chromosomes that are in contact and divide by the length of both chromosomes involved. The result is displayed in Figure~\ref{fig:chrom_contacts}. While a deeper comparison of these interaction strengths between cells is beyond the scope of this work, a simple visual analysis yielded no obvious relationships between the interaction strengths of the different cells. On a positive note, a certain resemblence can be seen to the same kind of image in (Stevens et al.\cite{stevens_3d_2017}) in Figure~1b, although a more detailed comparison could not be made due to a lack of the data from (Stevens et al.).

% section comparison_of_chromosomes_in_the_entire_genome_simulations (end)

\section{Simulation of individual chromosomes} % (fold)
\label{sec:simulation_of_individual_chromosomes}

Instead of simulating the entire genome, only individual chromosomes can be simulated in isolation. This can show how much of the structure of each chromosome is dependent on intrinsic interactions in the chromosome itself opposed to extrensic interactions with other chromosomes. Simulations were carried out for chromosome 1 and chromosome 19, as those are the largest and smallest chromosome respectively.

\subsection{Chromosome 1} % (fold)
\label{ssec:chromosome_1}

In Figure~\ref{fig:potential_energy_cell3_chrom1} the potential energy for the simulation of chromosome 1 of cell 3 is displayed. Compared to the potential energy of cell 2 in Figure~\ref{fig:potential_energy_cell2} the potential energies for the invdividual chromosome look a lot more spread out and unstable, but the coefficient of variation of the potential energy of \(1.24 \%\) is actually comparably low in reference to the coefficients of variation of the cell simulations as seen in Table~\ref{tab:simulation_pe_dists}. 

\begin{figure}[ht]
\centering
	\includegraphics[width=\figwidth]{potential_energy_cell3_chrom1.png}
	\caption{Potential energy of all frames for simulation of chromosome 1 of cell 3 indiviually.}
	\label{fig:potential_energy_cell3_chrom1}
\end{figure}

Looking at the RMSDs of the chromsome simulation though, seen in Figure~\ref{fig:potential_energy_cell3_chrom1}, the mean RMSD from frame 22, which is the lowest potential energy frame, is quite high at \(\num{4.6(10)}\). Even worse, even though frame 1 and frame 71 have a similarly low energy compared to frame 22, their respective RMSDs from frame 22 are still larger than \(3.0\), implying that neither of these frames represents a true ground state. The mean RMSDs from the respective lowest energy frame for the simulations of chromosome 1 for all cells in Table~\ref{tab:chrom1_mean_rmsds} show that this is the case for all cells. Unsurprisingly, when comparing these individually simulated chromosomes with their respective counterparts in the simulations of the entire genome, by calculating the RMSD from the chromosome in the average trajectory, the results are similarly bad. This implies that the structure of chromosome 1 when simulated individually is quite dissimilar to the structure when simulated in the context of the entire cell.

\begin{figure}[ht]
\centering
	\includegraphics[width=\figwidth]{rmsd_cell3_chrom1.png}
	\caption{RMSD of each frame of the simulation of chromosome 1 of cell 3 with the lowest-energy frame, frame 22.}
	\label{fig:rmsd_cell3_chrom1}
\end{figure}

\begin{table}[ht]
\centering
  \caption{Mean of RMSDs between each frame of chrosomome 1 simulation to lowest energy frame of this simulation and to chromosome 1 in the average trajectory of the entire cell simulation for each cell.}
  \label{tab:chrom1_mean_rmsds}
  \sisetup{ table-alignment-mode=none }
  \begin{tabular}{l @{\phantom{abc}} S S S S S S S S}
  \toprule
    Cell & 1 & 2 & 3 & 4 & 5 & 6 & 7 & 8 \\
  \midrule
    \parbox{4cm}{Mean RMSD to \\ lowest energy frame} & 4.6 & 3.4 & 4.6 & 2.5 & 4.2 & 3.2 & 4.9 & 8.2 \\
    Mean RMSD to avg cell & 4.6 & 3.9 & 5.6 & 3.1 & 4.6 & 6.8 & 5.1 & 7.8 \\
  % \midrule
  %   Internal contacts & 3735 & 2594 & 1231 & 3197 & 2323 & 2860 & 1599 & 1310 \\
  %   External contacts & 1562 & 660 & 854 & 384 & 849 & 294 & 300 & 349 \\
  %   \parbox{4cm}{\centering Proportion of \\ internal contacts} & 71\% & 80\% & 59\% & 89\% & 73\% & 91\% & 84\% & 79\% \\
  \bottomrule
  \end{tabular}
\end{table}

% subsection chromosome_1 (end)

\FloatBarrier

\subsection{Chromosome 19} % (fold)
\label{ssec:chromosome_19}

The same analysis has been repeated for chromosome 19, which is the shortest chromosome with only 584 beads compared to chromosome 1 with 1924 beads. For the RMSDs from the minimum energy frame, displayed in \ref{fig:rmsd_cell3_chrom19}, we see the same pattern as already for chromosome 1, with the RMSD being both very high and unstable. In particular no stable ground state configuration is reached again. The RMSDs to the average trajectory from the entire genome simulation can be seen in Table~\ref{tab:chrom19_mean_rmsds}. While most of the RMSDs are similarly high, for cell 1 and cell 4 the mean RMSD to the average simulated cell drops to \(2.3\) and \(1.5\) respectively, which is indicative of decent resemblance. Nevertheless, generally the difference between the individually simulated chromosomes and the chromosomes in the cell is still quite high, suggesting that the results of cell 1 and cell 4 are merely the exception from the rule and that the stabilising effect of other chromosomes is a non-neglegible factor for determining a chromosome's structure.

\begin{figure}[ht]
\centering
  \includegraphics[width=\figwidth]{rmsd_cell3_chrom19.png}
  \caption{RMSD of each frame of the simulation of chromosome 19 of cell 3 with the lowest-energy frame, frame 73.}
  \label{fig:rmsd_cell3_chrom19}
\end{figure}

\begin{table}[ht]
\centering
  \caption{Mean of RMSDs between each frame of chrosomome 19 simulation to lowest energy frame of this simulation and to chromosome 19 in the average trajectory of the entire cell simulation for each cell.}
  \label{tab:chrom19_mean_rmsds}
  \sisetup{ table-alignment-mode=none }
  \begin{tabular}{l @{\phantom{abc}} S S S S S S S S}
  \toprule
    Cell & 1 & 2 & 3 & 4 & 5 & 6 & 7 & 8 \\
  \midrule
    \parbox{4cm}{Mean RMSD to \\ lowest energy frame} & 1.8 & 3.7 & 4.9 & 1.2 & 3.6 & 2.5 & 4.6 & 5.9 \\
    Mean RMSD to avg cell & 2.3 & 4.1 & 6.0 & 1.5 & 4.4 & 3.1 & 4.8 & 6.1 \\
  % \midrule
  %   Internal contacts & 1229 & 717 & 243 & 1037 & 684 & 761 & 460 & 333 \\
  %   External contacts & 605 & 394 & 544 & 214 & 381 & 302 & 142 & 169 \\
  %   \parbox{4cm}{\centering Proportion of \\ internal contacts} & 67\% & 65\% & 31\% & 83\% & 64\% & 72\% & 76\% & 66\% \\
  \bottomrule
  \end{tabular}
\end{table}

% subsection chromosome_19 (end)

% section simulation_of_individual_chromosomes (end)

% One possible explanation for the dissimililarity betwen the chrosomosome structure in the cell and individually could be that the chromosome structure gets stabilised by its contacts with other chromosomes. To examine the strength of this effect the number of Hi-C contacts of the chromosome both with itself and and with other contacts and then calculate the ratio of internal contacts.

% chapter individual_chromosomes (end)